### DISCUSSION TITLE
操作面板找不到了...

### DISCUSSION THREAD

**Discussion by 19307130278** [the1812/Bilibili-Evolved#2731] [提问 / Q&A] [answered]
不懂就问，为啥把插件和包全安装之后找不到操作面板了？求问快捷显示它的方法

**Comment by 19307130278** [comment]
就是之前还用了一两天，今天把功能都装好发现它不见了

**Comment by Geeyun-JY3** [comment, marked as answer]
是不是装了 `自动隐藏侧栏`

**Reply by 19307130278** [reply]
有可能，列表的没看全装了...求问如何关掉>>>感谢

**Reply by Geeyun-JY3** [reply]
你应该善用搜索，之前 2.0-preview 刚出来还有 `我全都要` 这个合集包的时候就很多人有类似的情况：#2452、#2542

**Comment by 19307130278** [comment]
重装解决99.9%的问题
